dsnp/broadcast.spec.ts

```diff
@@ -0,0 +1,52 @@
+import fs from "fs";
+import { ParquetWriter } from "@dsnp/parquetjs";
+import { transformToParquetjs } from "../helpers/parquet";
+import broadcastSchema from "./broadcast";
+import * as generators from "@dsnp/test-generators";
+import { ParquetModel } from "../types/frequency";
+
+describe("Broadcast Spec", () => {
+  it("can build a parquet file", async () => {
+    await expect(async () => {
+      const [schema, bloom] = transformToParquetjs(broadcastSchema);
+      await ParquetWriter.openStream(
+        schema,
+        {
+          write: jest.fn(),
+          end: jest.fn(),
+        },
+        { bloomFilters: bloom }
+      );
+    }).not.toThrow();
+  });
+
+  it("Compressing all columns is best", async () => {
```

Comment:
What it says on the box. It leaves one column uncompressed each time and makes sure that all columns compressed is best. I was surprised, but it it true even for just 10 entries (although more pronounced at more.